### DISCUSSION TITLE
Problems following the quick start tutorial. prisma-generate not invoked

### DISCUSSION THREAD

**Discussion by canrozanes** [prisma/prisma#4781] [Q&A] [answered]
I followed the step by step tutorial [start-from-scratch-typescript-postgress](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-typescript-postgres#prerequisites) from the beginning and am stuck on the section **Install and generate Prisma Client** where installing `@prisma/client` does not trigger a  `prisma-generate `command and when I try to manually trigger it inside the` prisma/` directory, I get a `Couldn't find `prisma.yml` file. Are you in the right directory?` error. 

What am I missing? Is there a missing step in the tutorial? 

If anyone could direct me to right section in the docs to fix this, it would be much appreciated.

**Comment by ryands17** [comment, marked as answer]
Hey @canrozanes 👋

Have you installed `@prisma/cli` locally? If not, then do install it locally and the `@prisma/cli` and `@prisma/client` versions should match.

After you have installed it locally, you do not need to enter the `prisma/` directory to run commands, you can simply do it from your project root. Just running the following should work fine:

```
npx prisma generate
```

Let me know if this works :)

**Reply by canrozanes** [reply]
that worked thank you so much!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
